Sources/SkyFloatingLabelTextFieldWithIcon.swift

```diff
@@ -12,9 +12,68 @@
 import UIKit
 
 /**
+ Enum to identify the type of icon
+ */
+public enum IconType: Int {
+
+    case font
+    case image
+
+}
+
+/**
  A beautiful and flexible textfield implementation with support for icon, title label, error message and placeholder.
  */
 open class SkyFloatingLabelTextFieldWithIcon: SkyFloatingLabelTextField {
+    
+    /// A enum determining if user wants to use iconFont or iconImage
+    
+    @IBInspectable
+    var iconTypeAdapter:Int {
+        
```

Comment:
Get rid of all the redundant new lines here, but leave one line between set and get